src/js/edu-benefits/utils/helpers.jsx

```diff
@@ -142,3 +142,7 @@ export const preferredContactMethodLabels = {
   phone: 'Phone'
 };
 
+export const relationshipLabels = {
+  spouse: 'Spouse',
+  child: 'Child'
```

Comment:
If these are the ones in the 5490, there was a copy change; they should be:

```js
export const relationshipLabels = {
  child: 'Child/Stepchild/Adopted Child',
  spouse: 'Spouse/Surviving Spouse',
};
```

I don't know if it would be better to do that here or to do that when resolving the merge conflict, but it might be less likely to be lost if done here.